### PR TITLE
Allow setting samesite cookie

### PR DESCRIPTION
**Pull request by 6543**
source: https://gitea.com/macaron/i18n/pulls/7

### CODE REVIEW

**Comment by 6543** [comment]
@unknwon this is ready :)

File: i18n.go [hunk 1]
```diff
@@ -106,6 +108,8 @@ type Options struct {
 	Secure bool
 	// Set the HTTP Only flag on the `lang` cookie. Default is disabled.
 	CookieHttpOnly bool
+	// Set the SameSite flag on the `lang` cookie. Default is unset
```
Comment (unknwon) [inline i18n.go:111]:
```suggestion
	// Set the SameSite flag on the `lang` cookie. Default is unset.
```

**Review by unknwon — APPROVED** [review]
Thanks!

**Comment by 6543** [comment]
@unknwon updated

**Comment by unknwon** [comment]
https://github.com/go-macaron/i18n/releases/tag/v0.6.0 has been published for this merge.

**Pull request merged**